inc/Smartling/WP/View/ConfigurationProfiles.php

```diff
@@ -263,6 +266,12 @@ class="toggleExpert"><strong><?= __('Show Expert Settings'); ?></strong></a>
                             ?>
                         </td>
                     </tr>
+                    <tr>
+                        <th><label for="<?= GlobalSettingsManager::SETTING_CUSTOM_DIRECTIVES?>"><a href="https://help.smartling.com/hc/en-us/articles/360008000893-XML#directives"><?= __('Custom directives.')?></a><?= __('XML processor will wrap each line from the text area in') ?><pre>&lt;!-- smartling.%%line%% --&gt;</pre>. <?= __('Invalid directives WILL BREAK PROCESSING.')?></label></th>
```

Comment:
Added space after period. Changed text.
```suggestion
                        <th><label for="<?= GlobalSettingsManager::SETTING_CUSTOM_DIRECTIVES?>"><a href="https://help.smartling.com/hc/en-us/articles/360008000893-XML#directives"><?= __('Custom directives')?></a><?= __('. The connector will wrap each line from this text area into inline directives') ?><pre>&lt;!-- smartling.%%line%% --&gt;</pre><br><?= __('Invalid directives WILL BREAK PROCESSING.')?></label></th>
```